packages/thirdweb/src/react/core/providers/thirdweb-provider.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { describe, it } from "vitest";
+import { render } from "~test/react-render.js";
+import { ThirdwebProvider } from "./thirdweb-provider.js";
+
+describe("thirdweb-provider", () => {
```

Comment:
_Technically_ every test does this because in `test/react-render.js` we wrap every `render` call in our Provider. So this test is actually testing if a provider within a provider will work.